Plugin: 我要一张xx涩图

**插件名称：**

我要一张xx涩图

**插件功能：**

从多个api获取色图并根据场景整合的色图插件

**PyPI 项目名：**

nonebot-plugin-setu-collection

**插件 import 包名：**

nonebot_plugin_setu_collection

**插件项目仓库/主页链接：**

https://github.com/KarisAya/nonebot_plugin_setu_collection

**标签：**

[{"label":"LoliconAPI","color":"#5adba8"},{"label":"色图","color":"#7ab2e1"}]

# 📃 商店发布检查结果

> Plugin: 我要一张xx涩图

**⚠️ 在发布检查过程中，我们发现以下问题：**
<pre><code><li>⚠️ 插件加载测试未通过。<details><summary>测试输出</summary>插件 nonebot_plugin_setu_collection 的信息如下：
    name         : nonebot-plugin-setu-collection 
     version      : 0.0.1                          
     description  : 从多个api获取色图并根据场景整合的色图插件         
    
    dependencies
     - nonebot-adapter-onebot *
     - nonebot2 *
     - requests *
插件 nonebot_plugin_setu_collection 加载出错：
02-02 12:22:14 [SUCCESS] nonebot | NoneBot is initializing...
02-02 12:22:14 [INFO] nonebot | Current Env: prod
02-02 12:22:14 [ERROR] nonebot | Failed to import &quot;nonebot_plugin_setu_collection&quot;
Traceback (most recent call last):
  File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
    valid = load_plugin(&quot;nonebot_plugin_setu_collection&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
&gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
    module = importlib.import_module(name)
  File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1006, in _find_and_load_unlocked
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 688, in _load_unlocked
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 233, in exec_module
    super().exec_module(module)
  File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 883, in exec_module
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 241, in _call_with_frames_removed
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_setu_collection/__init__.py&quot;, line 24, in &lt;module&gt;
    from .utils import MirlKoi,Anosu,Lolicon,is_MirlKoi_tag
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_setu_collection/utils.py&quot;, line 1, in &lt;module&gt;
    from .api.MirlKoi import MirlKoi,is_MirlKoi_tag
ModuleNotFoundError: No module named &#x27;nonebot_plugin_setu_collection.api&#x27;</details></li></code></pre>
<details><summary>详情</summary><pre><code><li>✅ 标签: LoliconAPI-#5adba8, 色图-#7ab2e1。</li><li>✅ 项目 <a href="https://github.com/KarisAya/nonebot_plugin_setu_collection">主页</a> 返回状态码 200。</li><li>✅ 包 <a href="https://pypi.org/project/nonebot_plugin_setu_collection/">nonebot_plugin_setu_collection</a> 已发布至 PyPI。</li></code></pre></details>

---

💡 如需修改信息，请直接修改 issue，机器人会自动更新检查结果。
💡 当插件加载测试失败时，请发布新版本后在当前页面下评论任意内容以触发测试。

💪 Powered by [NoneBot2 Publish Bot](https://github.com/nonebot/nonebot2-publish-bot)
<!-- PUBLISH_BOT -->

--- a/website/static/plugins.json
+++ b/website/static/plugins.json
@@ -4113,5 +4113,24 @@
       }
     ],
     "is_official": false
+  },
+  {
+    "module_name": "nonebot_plugin_setu_collection",
+    "project_link": "nonebot-plugin-setu-collection",
+    "name": "我要一张xx涩图",
+    "desc": "从多个api获取色图并根据场景整合的色图插件",
+    "author": "KarisAya",
+    "homepage": "https://github.com/KarisAya/nonebot_plugin_setu_collection",
+    "tags": [
+      {
+        "label": "LoliconAPI",
+        "color": "#5adba8"
+      },
+      {
+        "label": "色图",
+        "color": "#7ab2e1"
+      }
+    ],
+    "is_official": false
   }
 ]

